docs/en/base/forms.md

```diff
@@ -92,5 +92,5 @@ You can use the ```<fieldset>``` element to divide the form into different logic
 
 <hr />
 
-Related:
+## Related:
```

Comment:
I believe the related section is supplementary content. Therefore the title should be a `h3`. 

For example:
```
### Related:
```